CWE/events/East Turkestan Additions.txt

```diff
@@ -0,0 +1,1404 @@
+			consciousness = 3
+		}
+	}
+}
+
+
+#1947 unrest
+country_event = {
+	id = 8023808
+	title = "1947 unrest"
+	desc = EVTDESC8023808
+	picture = "ets_rally2"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = XIJ
+		year = 1947
+		NOT = { year = 1950 }
+
+		war = no
+		is_vassal = no
+
+		NOT = { has_accepted_culture = uigher }
+
+		ETS = { 
+			exists = no
+		}
+		KMT = { war_with = CHI } #Chinese Civil War ongoing
+
+	}
+
+	mean_time_to_happen = { months = 4 }
+
+	option = {
+		name = "Introduce troops to stablize the situation."
+
+		1600 = { #Urumqi
+			add_province_modifier = { name = nationalist_agitation duration = 380 }
+			any_pop = {
+				limit = { culture = uighur }
+				ideology = { factor = 0.10 value = communist }
+				militancy = 8
+				consciousness = 8
+			}
+		}
+		1596 = { #Altay
+			add_province_modifier = { name = nationalist_agitation duration = 380 }
+			any_pop = {
+				limit = { culture = uighur }
+				ideology = { factor = 0.10 value = communist }
+				militancy = 8
+				consciousness = 8
+			}
+		}
+		1595 = { #Gulja
+			add_province_modifier = { name = nationalist_agitation duration = 380 }
+			any_pop = {
+				limit = { culture = uighur }
+				ideology = { factor = 0.10 value = communist }
+				militancy = 8
+				consciousness = 8
+			}
+		}
+
+		3373 = { #Turpan
+			any_pop = {
+				limit = { culture = uighur }
+				ideology = { factor = 0.10 value = nationalist }
+				militancy = 3
+				consciousness = 3
+			}
+		}
+		1599 = { #Kumul
+			any_pop = {
+				limit = { culture = uighur }
+				ideology = { factor = 0.10 value = nationalist }
+				militancy = 3
+				consciousness = 3
+			}
+		}
+	}
+}
+
+#1948 massacre in Turfan
+country_event = {
+	id = 8023809
+	title = "Turfan Massacre"
+	desc = EVTDESC8023809
+	picture = "ets_rally2"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = XIJ
+		year = 1948
+		NOT = { year = 1950 }
+
+		war = no
+		is_vassal = no
+
+		NOT = { has_accepted_culture = uigher }
+
+
+		ETS = { 
+			exists = no
+		}
+		KMT = { war_with = CHI } #Chinese Civil War ongoing
+
+	}
+
+	mean_time_to_happen = { months = 4 }
+
+	option = {
+		name = "Things are getting more chaotic..."
+		badboy = 3
+		3373 = { #Turpan
+			any_pop = {
+				limit = { culture = uighur }
+				reduce_pop = 0.99
+				ideology = { factor = 0.10 value = nationalist }
+				militancy = 3
+				consciousness = 3
+			}
+		}
+	}
+}
+
+
+#Beitashan Incident
+country_event = {
+	id = 8023810
+	title = "Beitashan Incident"
+	desc = EVTDESC8023810
+	picture = "baitag"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = XIJ
+		year = 1948
+		NOT = { year = 1950 }
+
+		is_vassal = no
+		war = no
+
+		NOT = { has_accepted_culture = uigher }
+
+		ETS = { 
+			exists = no
+		}
+
+		KMT = { war_with = CHI } #Chinese Civil War ongoing
+
+	}
+
+	mean_time_to_happen = { months = 2 }
+
+	option = {
+		name = "Attack Soviet and Mongolian positions to try to quiet the north."
+		badboy = 2
+		diplomatic_relation = { who = MGL value = -200 }

```

Comment:
random_country = { limit = { tag = MGL GOVTYPE is_vassal = no } relation = { who = THIS value = -200 } }